### ISSUE TITLE
[Bug] LogView crashes with undefined host

### ISSUE THREAD

**Issue by SegFaultAX** [riemann/riemann-dash#70]
tl;dr events with nil hosts in Riemann cause LogViews in riemann-dash to die at template-rendering time.

Here's an example stream (from the default riemann.config) that will emit events without a host:

``` clojure
(with {:metric 1 :host nil :state "ok" :service "events/sec"}
  (rate 5 index))
```

When these events are streamed to riemann-dash, the `host` key will not be present in the event object. Rendering the underscore template ([here](https://github.com/aphyr/riemann-dash/blob/master/lib/riemann/dash/public/views/log.js#L78)) without all required keys will result in a template exception.

The quick and dirty fix is to merge the event object with an object containing default values (empty strings?) for the following keys before rendering the template:
1. host
2. service
3. state
4. metric
5. description
